src/main/java/ru/skypro/homework/dto/AdsResponseDTO.java

```diff
@@ -0,0 +1,17 @@
+package ru.skypro.homework.dto;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Getter
+@Setter
+public class AdsResponseDTO {
+    private Integer count;
+    private List<AdResponseDTO> results = Collections.singletonList(new AdResponseDTO());
```

Comment:
Инициализация излишне. Все равно коллекция будет извне задаваться
```suggestion
    private List<AdResponseDTO> results;
```